network/network.go

```diff
@@ -174,8 +176,12 @@ func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 	}
 
 	var wg sync.WaitGroup
-
-	pods, err := c.k8sClient.CoreV1().Pods(c.namespace).List(ctx, metav1.ListOptions{LabelSelector: "app=net-exporter"})
+	
+    opts := metav1.ListOptions{
+		LabelSelector: fmt.Sprintf( "app=%s", appName ),
+	}
+	
+	pods, err := c.k8sClient.CoreV1().Pods(c.namespace).List(ctx, opts)
```

Comment:
Nit: whitespace is weird here

Reply:
The original code uses tabs, I'm using code server which is putting spaces by default. Just to make things interesting..

Reply:
I think I got it this time :)